cli.py

```diff
@@ -69,6 +76,12 @@ def remute_channel(channel_id):
         sys.stderr.write("Please set the SLACK_TOKEN environment variable.\n")
```

Comment:
Can we make it this?

    if TOKEN is None or TOKEN == "":

Reply:
Done